### ISSUE TITLE
Postgresql link is broken

### ISSUE THREAD

**Issue by Steveorevo** [Steveorevo/hestiacp-dockered#1]
Currently, the Postgresql database admin (aka phpPgAdmin) at https://hestiacp.dev.cc/phppgadmin/ displays the following error message:
```
Configuration error: Copy conf/config.inc.php-dist to conf/config.inc.php and edit appropriately.
```

Checking the systemd service via `service --status-all` indicates the postgresql service is installed and running.

**Comment by jaapmarcus** [comment]
Don't we inlcude the config.php in  https://github.com/hestiacp/hestiacp/blob/main/install/deb/pga/config.inc.php

Maybe it doesn't include them?

**Comment by Steveorevo** [comment]
Fixed in fdd626eba96fe8d671e960069f866d58a04fedbd
